test/e2e/self_hosted.go

```diff
@@ -226,6 +226,26 @@ func SelfHostedSpec(ctx context.Context, inputGetter func() SelfHostedSpecInput)
 			Namespace:            namespace.Name,
 		})
 
+		// Note: clusterctl should restore the managedFields to the same as before the move,
+		// and thus removing any managedField entries with clusterctl as a manager. This should happen
+		// for all the objects processed by move, but for sake of simplicity we test only the Cluster
+		// object. The Cluster object has special processing for the paused filed during the move to
```

Comment:
```suggestion
		// object. The Cluster object has special processing for the paused field during the move to
```
nit: otherwise lgtm

Reply:
Fixed that nit myself as I don't want to block this PR from getting into v1.3 rc.0 (and thus the release-1.3 branch) because of that